src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,84 @@
+.calendar {
+  margin: 200px auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  gap: $gap-size;
+  width: area-size(7);
+  border: 1px solid lightgray;
+  border-radius: $area-border-radius;
+  box-sizing: content-box;
+  padding: 10px;
+  background-color: lightgray;
+  perspective: 500px;
+
+  &__day {
+    @include square($day-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid grey;
+    border-radius: $cell-border-radius;
+    background-color: #eee;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink !important;
```

Comment:
Avoid using !important as it can make your CSS difficult to manage and maintain. Consider using a more specific selector or adjusting the order of your rules to increase specificity.

Reply:
it was so easy way to complete task faster... ok :)